content/docs/gke/customizing-gke.md

```diff
@@ -7,56 +7,104 @@ weight = 2
 This guide describes how to customize your deployment of Kubeflow on Google 
 Kubernetes Engine (GKE) in Google Cloud Platform (GCP).
 
+## Customizing Kubeflow before deployment
+
+The Kubeflow deployment process is divided into two steps, **build** and 
+**apply**, so that you can modify your configuration before deploying your 
+Kubeflow cluster.
+
+Follow the guide to [deploying Kubeflow on GCP](/docs/gke/deploy/deploy-cli/).
+When you reach the 
+[setup and deploy step](/docs/gke/deploy/deploy-cli/#set-up-and-deploy),
+**skip the `kfctl apply` command** and run the **`kfctl build`** command 
+instead, as  described in that step. Now you can edit the configuration files 
+before deploying Kubeflow.
+
+## Customizing an existing deployment
+
+You can also customize an existing Kubeflow deployment. In that case, this 
+guide assumes that you have already followed the guide to 
+[deploying Kubeflow on GCP](/docs/gke/deploy/deploy-cli/) and have deployed
+Kubeflow to a GKE cluster.
+
 ## Before you start
 
-This guide assumes you have already set up Kubeflow with GKE. If you haven't done
-so, follow the guide to [deploying Kubeflow on GCP](/docs/gke/deploy/).
+This guide assumes the following settings: 
 
-## Customizing Kubeflow
+* The `${KFAPP}` environment variable contains the name
+  of your Kubeflow application directory, as described in the guide to
+  [deploying Kubeflow on GCP](/docs/gke/deploy/deploy-cli/):
 
-You can use [kustomize](https://kustomize.io/) to customize Kubeflow.
+  ```
+  export KFAPP=<your choice of application directory name>
+  ```
+
+* The `${CONFIG}` environment variable contains the path to a kfctl 
+  configuration file, as described in the section on [preparing your environment 
+  for deployment](/docs/gke/deploy/deploy-cli/#prepare-environment):
 
-The deployment process is divided into two steps, **generate** and **apply**, so that you can
-modify your deployment before actually deploying.
+  ```
+  export CONFIG="{{% config-uri-gcp-iap %}}"
+  ```
+
+    Or:
 
-To customize GCP resources (such as your Kubernetes Engine cluster), you can modify the deployment manager configs in **${KFAPP}/gcp_config**.
+  ```
+  export CONFIG="{{% config-uri-gcp-basic-auth %}}"
+  ```
+
+* All commands below start from the parent directory that contains your 
+  `${KFAPP}` directory:
+
+  ```
+  cd <the parent directory of your KFAPP directory>
```

Comment:
I think people have gotten confused about absolute directory names vs. app names etc...

Would it help if we wrote this way

```
export APPSDIR=<Directory where you'd like your Kubeflow deployments to be stored>
export KFNAME=<Name for your kubeflow deployment>
export KFAPPDIR=${APPSDIR}/${KFNAME}

mkdir -p ${KFAPPDIR}
cd ${KFDAPPDIR}
```

Reply:
Yes, that's a good idea. I'll hive that off to a separate issue, because it means wide-spread updates throughout the docs. Many docs rely on the semantics of `${KFAPP)`. I've created this issue: https://github.com/kubeflow/website/issues/1252

Let's do this outside the scope of a particular release - so, when the docs for Kubeflow v0.7 are safely bedded down.